core/src/main/resources/lib/hudson/executors.jelly

```diff
@@ -39,8 +39,9 @@ THE SOFTWARE.
       <j:if test="${!c.acceptingTasks}"> <st:nbsp/> (${%suspended})</j:if>
     </d:tag>
     <d:tag name="executor">
+      <j:set var="ae" value="${e.asynchronousExecution}"/>
```

Comment:
I doubt that should have any relationship. If `asynchronousExecution` is set, the executor has already started, so any queue changes are long done. This is only displaying a snapshot of the executor state anyway.